pkg/route/hostassignment/assignment_test.go

```diff
@@ -52,6 +53,13 @@ func TestHostWithWildcardPolicies(t *testing.T) {
 		expected          string
 		expectedSubdomain string
 
+		// fields for externalCertificate
+		validType  bool
+		secretData map[string]string
+		validNS    bool
```

Comment:
Where are these fields used?

Reply:
lol :)  those were not used anywhere. Removed, thanks